worldengine/generation.py

```diff
@@ -105,12 +105,20 @@ def initialize_ocean_and_thresholds(world, ocean_level=1.0):
     """
     e = world.elevation['data']
     ocean = fill_ocean(e, ocean_level)
-    hl = find_threshold_f(e, 0.10)  # the highest 10% of all (!) land are declared hills
-    ml = find_threshold_f(e, 0.03)  # the highest 3% are declared mountains
+
+    # values chosen to emulate the exponential behavior seen here:
+    # http://www.ngdc.noaa.gov/mgg/global/etopo1_surface_histogram.html
+    pl = find_threshold_f(e, 0.95, ocean=ocean)  # the highest x% of land are declared plains; save some for beaches (?)
+    hl = find_threshold_f(e, 0.50, ocean=ocean)  # the highest x% are declared hills
```

Comment:
I just checked something: Look at [the first image](https://github.com/Mindwerks/worldengine-data/pull/17/files) in swipe-mode.
Doesn't really tell anything about the quality of the chosen values, only that I should check if the ancient maps need a little tweak to make use of the new levels.

The values themselves seem pretty good to me; the old ones seem much more sketchy, with hills being the substitute for a proper mountain-level. Once we agree on good values, I will check the three places that actually make use of them (river-generation, temperature-generation and ancient map-generation). I think there is some detail to be gained in those. :)